.github/workflows/builds.yml

```diff
@@ -1518,14 +1518,15 @@ jobs:
                 libyaml-dev \
                 libcap-ng-dev \
                 libcap-ng0 \
+                liblua5.4-dev \
```

Comment:
This package is listed twice.

Reply:
Oops.  Doesn't break anything but easy to fix